cypress/support/hub-commands.ts

```diff
@@ -319,7 +319,7 @@ Cypress.Commands.add(
         formData.append('file', Cypress.Blob.binaryStringToBlob(fileData), filePath);
 
         cy.hubPostRequest({
-          url: hubAPI`/v3/plugin/ansible/content/staging/collections/artifacts/`,
+          url: hubAPI`/v3/plugin/ansible/content/${repository || 'validated'}/collections/artifacts/`,
```

Comment:
Slightly worried about `staging` not being the default since that's where collections go when uploaded the normal way.
But .. I suppose most tests just care about having an approved collection, and this achieves that :+1: